release.yml

```diff
@@ -1,4 +1,4 @@
-version: 2.2.10
+version: 2.2.11
 
 # JIRA Project where the release issue should be created
 issue:
```

Comment:
This needs to be removed. The `issue` stanza doesn't exist anymore, `project` field is now top-level.
/me needs to document the format!